tutorials/python/python-video-generation/code/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+pylint = "*"
+autopep8 = "*"
+
+[packages]
+moviepy = "1.0.0"
+gizeh = "0.1.11"
```

Comment:
@Larox I'm not sure if that's the correct syntax for setting a version here. Could you please double check that `pipenv install` still works?

Reply:
You are right, I missed the `==` in the version. Updating it :)